avaframe/ana3AIMEC/aimecTools.py

```diff
@@ -134,6 +134,39 @@ def fetchReferenceSimNo(pathDict, cfgSetup):
     return pathDict
 
 
+def computeCellSizeSL(cfgSetup, refResulHeader):
+    """ Get the new (s, l) coordinate cell size
+        read by default the reference result file cell size.
```

Comment:
consider calling it refResultHeader - one more character but then it uses the proper name..